docs/supported-libraries.md

```diff
@@ -104,6 +104,7 @@ These are the supported libraries and frameworks:
 | [Micrometer](https://micrometer.io/)                                                                                                        | 1.5+ (disabled by default)         | [opentelemetry-micrometer-1.5](../instrumentation/micrometer/micrometer-1.5/library)                                                                                                                                                                                                                                                                                                    | none                                                                                                                                    |
 | [MongoDB Driver](https://mongodb.github.io/mongo-java-driver/)                                                                              | 3.1+                               | [opentelemetry-mongo-3.1](../instrumentation/mongo/mongo-3.1/library)                                                                                                                                                                                                                                                                                                                   | [Database Client Spans], [Database Client Metrics]&nbsp;[6]                                                                             |
 | [MyBatis](https://mybatis.org/mybatis-3/)                                                                                                   | 3.2+                               | N/A                                                                                                                                                                                                                                                                                                                                                                                     | none                                                                                                                                    |
+| [Nacos Client](https://nacos.io/)                                                                                                           | 2.0.0+                             | N/A                                                                                                                                                                                                                                                                                                                                                                                     | none                                                                                                                                    |
```

Comment:
could you put a note in here that indicates it's disabled by default? similar to how Micrometer does it